fluentgen/src/test/resources/code-model-fluentnamer-nulloperationgroup.yaml

```diff
@@ -0,0 +1,7298 @@
+&id005 !!com.azure.autorest.extension.base.model.codemodel.CodeModel
```

Comment:
6000+ yaml only to test nulloperationgroup?

Can you make a small swagger with 2 or 3 operation and use that for test?

Reply:
Thanks for reminding.. I found out that this file is actually the output of `FluentNamer` which is useless... I'll find another way to test it in `FluentNamer`. Logged in the new issue.